scopes/react/modules/webpack/fragments/style-preset/style-preset.ts

```diff
@@ -0,0 +1,192 @@
+import type { RuleSetRule } from 'webpack';
+import type { Options as SassLoaderOptions } from 'sass-loader';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import getCSSModuleLocalIdent from 'react-dev-utils/getCSSModuleLocalIdent';
+import { allCssRegex, cssRegex, sassRegex, lessRegex } from '@teambit/webpack.modules.style-regexps';
+
+// all deps are rather small (2kb - 50kb)
+const styleLoaderPath = require.resolve('style-loader');
+// * no need for sourceMaps - loaders respect `devTools` (preferably, 'inline-source-map')
+// * miniCssExtractPlugin - (somehow) breaks hot reloading, so using style-loader in dev mode
+
+export type CssOptions = {
+  /**
+   * type of style injecctor to use.
+   * style-loader has better hot reloading support but is inline,
+   * mini-css-extract-plugin actually creates css files
+   */
+  styleInjector?: 'mini-css-extract-plugin' | 'style-loader';
+  /**
+   * handle *.module.* as css-modules
+   * @default true
+   */
+  modules?: boolean;
+
+  /** override specific path to css-loader */
+  cssLoader?: string;
+
+  /** override specific path to postcss-loader */
+  postcssLoader?: string;
+  /** override css loader options */
+  cssLoaderOptions?: any;
+  /** override css injector options */
+  injectorOptions?: any;
+  /** options for postcss-loader (skipped if undefined) */
+  postcssOptions?: any;
+};
+
+export type CssDialectOptions = {
+  /** override specific path to resolve-url-loader */
+  resolveUrlLoader?: string;
+  /** override specific path to sass-loader */
+  sassLoader?: string;
+  /** override specific path to less-loader */
+  lessLoader?: string;
+  /** override sass loader options */
+  sassLoaderOptions?: SassLoaderOptions;
+  /** override less loader options */
+  lessLoaderOptions?: any;
+  /** override resolveUrl loader options */
+  resolveUrlOptions?: any;
+};
+
+export type StyleLoadersOptions = CssOptions & CssDialectOptions;
+
+export const cssLoaders = ({
```

Comment:
this should be names styles loaders not CSS loaders

Reply:
done